### PR TITLE
Refactor the XSLT module API doc

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/index.json [hunk 1]
```diff
@@ -616,6 +616,10 @@
             {
                 "name": "Task Service Timer",
                 "url": "task-service-timer"
+            },
+            {
+                "name": "XSLT Transformation",
+                "url": "xslt_transformation"
```
Comment (chamil321) [inline examples/index.json:622]:
```suggestion
                "url": "xslt-transformation"
```

File: examples/xslt-transformer/xslt_transformation.bal [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import ballerina/io;
```
Comment (chamil321) [inline examples/xslt-transformer/xslt_transformation.bal:1]: filename should be xslt-transformation.bal right?
Shall we change other occurrences as well?

File: examples/xslt-transformer/xslt_transformation.out [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and run the `ballerina run` command.
+$ ballerina run xslt_transformation.bal
```
Comment (chamil321) [inline examples/xslt-transformer/xslt_transformation.out:3]:
```suggestion
$ ballerina run xslt-transformation.bal
```

File: examples/xslt-transformer/xslt_transformation.bal (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply examples/xslt-transformer/xslt_transformation.bal]: No, folder name should be in hyphen but underscore should be in filename.

File: examples/xslt-transformation/xslt_transformation.description [hunk 4]
```diff
@@ -0,0 +1 @@
+// This example demonstrates how the XML content can be transform to HTML using a given XSL transformations.
```
Comment (praneesha) [inline examples/xslt-transformation/xslt_transformation.description:1]:
```suggestion
// This example demonstrates how the XML content can be transformed to HTML using a given XSL transformation.
```

File: stdlib/xslt/src/main/ballerina/src/xslt/Module.md [hunk 5]
```diff
@@ -1,72 +1,3 @@
...
-                                    </tr>
-                                </xsl:for-each>
-                                </table>
-                            </body>
-                        </html>
-                    </xsl:template>
-                </xsl:stylesheet>`;
-}
-```
+For information on the operation, which you can perform with this module, see the below Function. For examples on the usage of the operation see [XSLT Transformation](https://ballerina.io/v1-2/learn/by-example/xslt-transformer.html).
```
Comment (praneesha) [inline stdlib/xslt/src/main/ballerina/src/xslt/Module.md:3]:
```suggestion
For information on the operation, which you can perform with this module, see the below Function. For examples on the usage of the operation, see [XSLT Transformation](https://ballerina.io/v1-2/learn/by-example/xslt-transformer.html).
```

File: stdlib/xslt/src/main/ballerina/src/xslt/natives.bal [hunk 6]
```diff
@@ -17,11 +17,15 @@
 import ballerina/java;
 
 # Transforms the single-rooted XML content to another XML/HTML/plain text using XSL transformations.
+# ```ballerina
+# xml|error target = xslt:transform(sourceXml, xsl);
+# io:println(target);
```
Comment (chamil321) [inline stdlib/xslt/src/main/ballerina/src/xslt/natives.bal:22]: Shall we remove this line? it does not relate to the XSLT functionality

**Pull request merged**